### PR TITLE
18 - Modify PHY structs to use unconstrained types

### PR DESCRIPTION
**Pull request by yogwoggf**
Changes are in the title, there's not much to add.

This adds Windows support. It's also a blocker for issue #18 in the TASBox repo.

### CODE REVIEW

File: src/phy.cpp [hunk 1]
```diff
@@ -98,7 +98,7 @@ namespace PhyParser {
     std::vector<uint16_t> indices;
     indices.reserve(ledge.trianglesCount * 3);
 
-    uint16_t maxVertexIndex = 0;
+    unsigned int maxVertexIndex = 0;
```
Comment (Derpius) [inline src/phy.cpp]: Why was this changed?

**Review by Derpius — COMMENTED** [review]
I’m not sure how this fixes things, or that it wouldn’t be fixed by using (u)int32_t as IIRC that’s identical to (unsigned) int

File: src/phy.cpp (hunk 1 above)
Comment (yogwoggf) [reply src/phy.cpp]: To avoid narrowing errors in the proceeding `std::max` call below this line

**Comment by yogwoggf** [comment]
> I’m not sure how this fixes things, or that it wouldn’t be fixed by using (u)int32_t as IIRC that’s identical to (unsigned) int

Yep. I wrote these fixes long ago and only just took the time to actually identify the problem. Somehow your platform (Linux) is allowing the old definitions to straddle across the three `uint16_t` primitives for the Edge struct and therefore you're getting the consistent 4 bytes. On my platform (Windows) my compiler is not allowing the code to straddle those primitives and injects padding into the structure such that I am getting a consistent 12 bytes (my IDE did not detect that). 

Anyways point is that I fixed the structs correctly and now it works fine on Windows, and *hopefully* it will work the same on Linux since none of this is UB anymore.

[here's a similar situation](https://stackoverflow.com/questions/35056179/c-size-of-structure-with-bit-fields)

**Review by Derpius — APPROVED** [review]
